util/topgen.py

```diff
@@ -866,7 +866,11 @@ def main():
         generate_xbars(completecfg, out_path)
 
     # All IPs are generated. Connect phase now
-    elab_intermodule(completecfg)
+    # Find {memory, module} <-> {xbar} connections first.
+    im.autoconnect(completecfg)
```

Comment:
does it make sense to call this fabric auto connect?
I'm guessing in the future, this will evolve into {memory, module, subsystem} <-> {xbar} right?

Reply:
Ah.. I should've consider the subsystem too... Let me think